src/pages/Profile/Index.vue

```diff
@@ -27,14 +28,23 @@ if (typeof route.params.id !== 'string') {
 }
 const authorID = route.params.id;
 const profileExists = ref<boolean>(false);
-
+const numFollowers = ref<number>(0);
+const numFollowing = ref<number>(0);
 const profile = computed(() => profilesStore.getProfile(authorID));
 
 useMeta({
 	title: profile.value.name ? `${profile.value.name as string} -  Blogchain` : `@${authorID as string} -  Blogchain`,
 	htmlAttrs: { lang: 'en', amp: true },
 });
 
+onBeforeMount(async () => {
+	const c = await useConnectionsStore().fetchConnections(authorID);
+	if (c) {
+		numFollowers.value = c?.followers.size;
```

Comment:
Can these be eliminated to use values from the store?

The reason is that `fetchConnections` will take time to load, and we're awaiting on that before mounting the component, 

If we use the value from store with a getter, we can asynchronously load that value while also displaying current value in store.